static/src/javascripts/projects/common/modules/navigation/new-header.js

```diff
@@ -265,49 +265,90 @@ const initiateUserAccountDropdown = (): void => {
         });
 };
 
+const attachKeyEvent = (
+    elem: HTMLElement,
+    eventID: number,
+    action: Function
```

Comment:
Flow docs [advise against using the `Function` type](https://flow.org/en/docs/types/functions/#toc-function-type) as it's almost as bad as using `any`. Can we replace with `() => void` or similar?